storybook/storybook-docs/src/typography.stories.mdx

```diff
@@ -1,84 +1,160 @@
+/>
+
+Elk tekstniveau heeft een minimale tekstgrootte (bij een vensterbreedte van 320 pixels) en een maximale (bij 1440 pixels).
+Tussen beide stijgt de tekst linear, met een knik op een extra breakpoint van 854 pixels.
+Eerst stijgt de grootte sneller, daarna langzamer.
+
+Op dit moment reageert de typografie nog op 1 breakpoint bij een vensterbreedte van 854 pixels.
+Nu het grid doorontwikkeld is, stemmen we de typografie daar weer op af.
+Dit aparte breakpoint komt dus snel te vervallen.
 
 In de tabel hieronder vind je deze waardes in pixels:
 
-| niveau                     |   0 |   1 |   2 |   3 |   4 |   5 |   6 |   7 |
+| Niveau                     |   0 |   1 |   2 |   3 |   4 |   5 |   6 |   7 |
 | :------------------------- | --: | --: | --: | --: | --: | --: | --: | --: |
 | minimale tekstgrootte      |  32 |  32 |  28 |  24 |  20 |  22 |  18 |  16 |
 | tekstgrootte op breakpoint |  64 |  48 |  36 |  29 |  23 |  26 |  21 |  17 |
 | maximale tekstgrootte      |  80 |  56 |  40 |  32 |  25 |  28 |  22 |  18 |
 | relatieve regelhoogte      | 1.1 | 1.2 | 1.3 | 1.4 | 1.4 | 1.5 | 1.6 | 1.6 |
 
+Tekst op niveau 5 is groter dan op niveau 4.
+Dat is omdat titel niveau 4 kleinere letters heeft dan paragrafen met grote letters.
+Het alternatief om Heading 4 tekstniveau 5 te geven leek ons nog verwarrender.
+
 Voorbeelden van componenten en hun tekstniveaus:
 
-| level | componenten                          |
-| ----: | :----------------------------------- |
-|     0 | Page heading                         |
-|     1 | Heading 1                            |
-|     2 | Heading 2                            |
-|     3 | Heading 3, Blockquote                |
-|     4 | Heading 4, Top Task Link             |
-|     5 | Paragraph large                      |
-|     6 | Paragraph, alle Links behalve inList |
-|     7 | Paragraph small, inList Link         |
+| Componenten                                          | Niveau |
+| :--------------------------------------------------- | -----: |
+| Page heading                                         |      0 |
+| Heading 1                                            |      1 |
+| Heading 2                                            |      2 |
+| Heading 3, Blockquote                                |      3 |
+| Heading 4, Top Task Link                             |      4 |
+| Large Paragraph, Alert Title                         |      5 |
+| Paragraph, Button, Link behalve InList               |      6 |
+| Small Paragraph, InList Link, Form Label, Breadcrumb |      7 |
 
-## Contrast
+## Accentuering
 
-Het contrast tussen tekst en achtergrondkleur moet voldoen aan [de laatste wettelijke toegankelijkheidseisen van de WCAG-niveau AA](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html).
+Er zijn verschillende manieren om tekst te accentueren.
+Hierin hebben we de volgende keuzes gemaakt.
 
-## Uitlijning
+### Gewicht
 
-Het aantal karakters op een regel is maximaal 80.
```

Comment:
Why did you remove this? It's kind of an important rule, especially with our huge font sizes.

Reply:
True, but in practice the huge font sizes probably mean you should have less than 80 characters on a line. We ran into this issue regularly with onderzoek.amsterdam.nl, the large Paragraph font size spanning 10 columns or so is hard to read, you have to swivel your head from left to right to much. 

I'd really like to test the readability of the different max chars on a line for the different font sizes we have. But until that time, we should at least set a maximum width imo. We could define a maximum amount of columns or something, but I think a maximum amount of chars is easier to work with.